loopchain/blockchain/blockchain.py

```diff
@@ -265,14 +283,14 @@ def __add_block(self, block: Block, vote: Vote = None):
             ObjectManager().channel_service.stop_leader_complain_timer()
 
             # start new epoch
-            ObjectManager().channel_service.block_manager.epoch = Epoch.new_epoch(block.header.height + 1)
+            ObjectManager().channel_service.block_manager.new_epoch()
```

Comment:
I think it is not proper location.
It will be executed during not only consensus but also block sync.
And also it will do in citizen node.

In my opinion block sync and citizen node do not need it.